qldbdriver/errors.go

```diff
@@ -32,6 +32,10 @@ type txnError struct {
 	isISE         bool
 }
 
+func (e *txnError) Error() string {
+	return e.err.Error()
```

Comment:
We don't need this since we're not really treating it as an error, and this would never be called.

Reply:
Removed.